versions.properties

```diff
@@ -43,6 +43,7 @@ net.sf.ehcache:ehcache-core=2.6.0
 saxonVersion=9.5.1-8
 net.sf.saxon:Saxon-EE=$saxonVersion
 abderaVersion=1.1.3
+org.asciidoctor:asciidoctorj-diagram=1.5.0
```

Comment:
Even if it did come before apache alphabetically, getting in between the abdera declarations and the abdera version number is just bad.